CHANGELOG.md

```diff
@@ -1,6 +1,37 @@
 # Acra ChangeLog
 
-## [0.92.0](https://github.com/cossacklabs/acra/releases/tag/0.92.0), March 01th 2021
+## [0.93.0](https://github.com/cossacklabs/acra/releases/tag/0.93.0), May 27th 2021
+
```

Comment:
```suggestion
This release brings type awareness which improves transparent encryption on AcraServer. Type awareness means that it's possible to tell AcraServer what are the original data types for fields. During decryption, AcraServer will convert decrypted fields to their original data types. No need to change client application code to work with "binary data".

It's also possible to choose a placeholder for each data field if its decryption failed. AcraServer can send a placeholder like "<encrypted data>" instead of decryption errors, making developers' and users' life easier.

```

Reply:
i'm not sure that placeholder is good word for this case. Placeholder means that it can be replaced with something. But here we just replace actual value with specified value. Maybe mask or hidding value for each data field?

Reply:
updated. placeholder -> default value @Lagovas